doc/spec_dataValidityPropagation.dox

```diff
@@ -0,0 +1,90 @@
+// put the namespace around the doxygen block so we don't have to give it all the time in the code to get links
+namespace ChimeraTK {
+/**
+
+\page spec_dataValidityPropagation Technical specification: data validity propagation Specification version V1.0RC2
+
+> **This is a release candidate in implementation**
+
+> **NOTICE FOR FUTURE RELEASES: AVOID CHANGING THE NUMBERING!** The tests refer to the sections, incl. links and unlinked references from tests or other parts of the specification. These break, or even worse become wrong, when they are not changed consistenty!
+
+1. General idea
+---------------
+
+- 1.1 In ApplicationCore each variable has a data validiy flag attached to it. DataValidity can be 'ok' or 'faulty'.
+- 1.2 This flag is automatically propagated: If one of the inputs of an ApplicationModule is faulty, the data validity of the module becomes faulty, which means
+all outputs of this module will automatically be flagged as faulty.
+      Fan-outs might be special cases (see 2.4).
+- 1.3 If a device is in error state, all variables which are read from it shall be marked as 'faulty'. This flag is then propagated through all the modules (via 1.1.2) so it shows up in the control system.
+- 1.4 The user code has the possibility to query the data validity of the module
+- 1.5 The user code has the possibility to set the data validity of the module to 'faulty'. However, the user code cannot actively set the module to 'ok' if one of the inputs is 'faulty'.
```

Comment:
```suggestion
- 1.5 The user code has the possibility to set the data validity of the module to 'faulty'. However, the user code cannot actively set the module to 'ok' if any of its inputs are 'faulty'.
```

Reply:
For me 'and of its inputs' refers to the user code. I changed it to "any of the module inputs".